iOS/Flows/Conversation/Threads/ThreadViewController.swift

```diff
@@ -145,35 +145,22 @@ class ThreadViewController: DiffableCollectionViewController<ConversationSection
 
     // MARK: Data Loading
 
-    override func getAllSections() -> [ConversationSection] {
-        if let channelId = self.parentMessage.cid {
-            let placeholderSection = ConversationSection(sectionID: channelId.description,
-                                                        parentMessageID: "placeholderSection")
-            return [placeholderSection,
-                    ConversationSection(sectionID: channelId.description,
-                                        parentMessageID: self.parentMessage.id)]
-        }
-
-        return []
+    override func getAllSections() -> [MessageSequenceSection] {
+        return [.topMessages, .bottomMessages]
```

Comment:
The thread VC will have two sections, but the to section won't contain any messages.